qiskit/quantum_info/operators/symplectic/pauli_list.py

```diff
@@ -0,0 +1 @@
+    r"""Symplectic representation of a list Pauli matrices.
```

Comment:
It looks an incomplete file.

Reply:
Ugh, I accidentally added this wip file when fixing conflicts with master